ckanext/ontario_theme/plugin.py

```diff
@@ -937,6 +937,11 @@ def update_config(self, config_):
 ckanext.ontario_theme:schemas/ontario_theme_group.json
 """
 
+def new_resource_publish(id, resource_id):
+    '''New page for submitting new resource for publication.
+    '''
+    pkg_dict = toolkit.get_action(u'package_show')(None, {u'id': id})
+    return render_template('/package/new_resource_publish.html', id=id, resource_id=resource_id, pkg_dict=pkg_dict)
```

Comment:
This is in between 2 classes, can we move it above the `OntarioThemeExternalPlugin` class, so the function is with all the other standalone functions?

Reply:
Sure. I'll move it under `def help()` since it is also for a new page